### ISSUE TITLE
Replace abandoned patchwork/utf8 dependency

### ISSUE THREAD

**Issue by f-schnabel** [isotope/core#2548]
It would be great to replace the [patchwork/utf8](https://packagist.org/packages/patchwork/utf8) dependency since it is abandoned.
It is only used four times in the [DC_ProductData](https://github.com/isotope/core/blob/46fb9d424e063ad914c1131fcb369a3478cd41bb/system/modules/isotope/drivers/DC_ProductData.php) file to sort some `$options_sorter`:
```php
        // Sort by option values
        uksort($options_sorter, array(Utf8::class, 'strnatcasecmp'));
```

There also exists the [strnatcasecmp](https://www.php.net/manual/en/function.strnatcasecmp.php) function directly in PHP, but I am unsure if it works the same for non ASCII characters (and if thats even relevant).

If this can simply be replaced then I can also create a PR for this.

**Comment by aschempp** [comment]
@leofeyer @ausi any idea what we replaced that with in Contao core?

**Comment by leofeyer** [comment]
Yes:

<img width="567" alt="" src="https://github.com/user-attachments/assets/cc3b70da-db25-492b-a625-8cc05a8a951e">

See https://github.com/contao/contao/blob/4.13/core-bundle/src/Resources/contao/drivers/DC_Table.php#L5351-L5362

**Comment by aschempp** [comment]
Fixed in 31bcc536e22980148bc71de4744653f350f8c881
